### PR TITLE
rollback multi-stage builds

### PR DESCRIPTION
**Pull request by tchughesiv**
Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ spec:
 
 ## Build rhel-based image for release
 
-Requires `cekit` v3.6+ and `rhpkg` -
+Requires `cekit` v3.1+ and `rhpkg` -
```
Comment (spolti) [inline README.md:164]: we can still using cekit 3.6, no need to downgrade it.

File: deploy/catalog_resources/module.yaml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 schema_version: 1
 
 name: catalog_resources
-version: "1.0"
```
Comment (spolti) [inline deploy/catalog_resources/module.yaml:4]: it can remains.

File: modules/kie-cloud-operator/module.yaml [hunk 3]
```diff
@@ -1,6 +1,5 @@
 schema_version: 1
 
-name: builder
-version: "1.0"
```
Comment (spolti) [inline modules/kie-cloud-operator/module.yaml:4]: can remains as well.

File: README.md (hunk 1 above)
Comment (tchughesiv) [reply README.md]: ehh... not so sure more changes wouldn't be required

Comment (tchughesiv) [reply README.md]: since i was rolling back, was easiest to use what we were using in 7.7.1

Comment (tchughesiv) [reply README.md]: once it's working well, ill test with 3.6.0

Comment (tchughesiv) [reply README.md]: experiencing errors when testing w/ cekit 3.6.0. given the time constraints, sticking w/ 3.2.x for now -
```
2020-07-02 17:27:16,279 cekit        INFO     Generating files for OSBS engine
2020-07-02 17:27:16,288 cekit        INFO     Initializing image descriptor...
2020-07-02 17:27:16,298 cekit        WARNING  No value found for 'name' in '{"path": "modules"}' artifact; using auto-generated value of 'modules'
2020-07-02 17:27:16,305 cekit        INFO     Preparing resource 'modules'
validation.invalid
 --- All found errors ---
["Cannot find required key 'version'. Path: ''"]
2020-07-02 17:27:16,307 cekit        ERROR    Cannot validate schema: Module
```

**Pull request merged**